sdk/nodejs/cmd/pulumi-language-nodejs/testdata/GetProgramDependencies/node-modules-package.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
Feels like this file could be much smaller -- do we need a real-world example here? Would be good to make this the minimal thing to trigger the behaviour we want. We can lose the curse words while we're at it :joy: